tests/integration/test_data_cli.py

```diff
@@ -701,9 +702,31 @@ def test_search_get_id_not_found(invoke, search_id):
     assert 'Error: {"message": "Error message"}\n' == result.output
 
 
+@respx.mock
+def test_search_delete(invoke, search_id, search_result):
+    delete_url = f'{TEST_SEARCHES_URL}/{search_id}'
+    mock_resp = httpx.Response(HTTPStatus.NO_CONTENT, json=search_result)
```

Comment:
Ohh, cool catch on the NO_CONTENT status code. This is the code the API returns?

Reply:
I just checked the api and it looks like it returns a 204 - is that what NO_CONTENT is? 
https://developers.planet.com/docs/apis/data/reference/#tag/Item-Search/operation/DeleteSearch

Reply:
204 is indeed NO_CONTENT! https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204